smdebug/tensorflow/keras.py

```diff
@@ -337,7 +337,7 @@ def _save_inputs(self, check_before_write=True):
         # TODO
         pass
 
-    def _add_metric(self, metric_name):
+    def _add_metric(self, metric_name, metric_value=None):
```

Comment:
Adding type annotations makes it clear whether a bool/str/tensor is expected.

Reply:
done in the latest commit